app/page.tsx

```diff
@@ -1,113 +1,18 @@
-import Image from 'next/image'
+import { client } from '@/sanity/lib/client';
 
-export default function Home() {
+export default async function Home() {
+  const projects = await client.fetch<any[]>(
+    `*[_type == "projects"]`
+  );
```

Comment:
Maybe put all these actions/calls in one file and simply call them here?
```javascript
const projects = await getProjects()
```

This is so to keep the UI clean from the core logic

Reply:
Good idea to define the actions early on and in one place. This will give me an opportunity to think about all the request I'd like to make from the cms

Reply:
I've set a couple actions in this PR. I will add more to match as needed as the needs expand